konig-data-app-common/src/main/java/io/konig/data/app/common/ExtentContainer.java

```diff
@@ -209,6 +217,43 @@ public boolean validateQueryParam(String key, String value, URI datatype ) throw
 	
 	public String escapeUtils(String value) {
 		return StringEscapeUtils.escapeHtml(StringEscapeUtils.escapeSql(StringEscapeUtils.escapeJavaScript(value)));
-	}
-
-}
+	}
+	
+	 boolean validateValue (String value )throws DataAppException{
+		 String  errorMsg = MessageFormat.format("IllegalArgumentException : Invalid  value : {0}",
+				 value);
+		  Pattern special = Pattern.compile ("[!@#$%&*()+=|<>?{}\\[\\]~]");
+		  Matcher hasSpecial = special.matcher(value);		
```

Comment:
You should set...

```
   value = value.toUpperCase();
```

and then check only for the upper case versions of the keywords.